static/js/profile.js

```diff
@@ -34,24 +34,25 @@ var vm = new Vue({
 
 function update_data(runs, user) {
             
-    vm.totalratedruns = runs.length;
-    vm.email = user[0]['email'];
-    if (user[0]['email_confirmed']) {
+    vm.totalratedruns = runs['total_prompts'];
+    vm.email = user['email'];
+    if (user['email_confirmed']) {
         vm.emailverified = "Yes";
     } else {
         vm.emailverified = "No";
     }
     
-    vm.user_name = user[0]['username'];
+    vm.user_name = user['username'];
 }
 
 async function get_data(usern) {
```

Comment:
This should be wrapped in `window.onload` rather than get called from the script. I think because the script is at the end it might be fine, but I'm not sure. Other option is to just switch it to vue

Reply:
While we're at it, let's change `un` argument in the profile endpoint to like `profile_name`